executor/simple.go

```diff
@@ -219,6 +219,7 @@ func (e *SimpleExec) executeAlterUser(s *ast.AlterUserStmt) error {
 		errMsg := "Operation ALTER USER failed for " + strings.Join(failedUsers, ",")
 		return terror.ClassExecutor.New(CodeCannotUser, errMsg)
 	}
+	domain.GetDomain(e.ctx).NotifyUpdatePrivilege(e.ctx)
```

Comment:
If we meet error here, add a log.

Reply:
NotifyUpdatePrivilege already log errors.